lib/components/Modal/Bottom/modal-bottom.stories.tsx

```diff
@@ -0,0 +1,476 @@
+            description:
+                "ID of the modal container. If it wasn't passed, the modal will try to find any container with 'modal-root' ID. If there is no container with 'modal-root' ID the modal will be insert inside document.body.",
+            control: { type: "string" },
+        },
+        primaryButtonLabel: {
+            table: { type: { summary: "ReactNode" } },
+            description: "Label for the primary button.",
+            control: { type: "string" },
+        },
+        primaryButtonCallback: {
+            table: { type: { summary: "(() => void) | undefined" } },
+            description:
+                "Function which will be called on clicking on primary button.",
+            control: { type: null },
+        },
+        secondaryButtonLabel: {
+            table: { type: { summary: "ReactNode | undefined" } },
+            description: "Label for the secondary button.",
+            control: { type: "string" },
+        },
+    },
+} satisfies Meta<typeof ModalBottom>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const DefaultModalBottom: Story = {
+    name: "Default Modal Bottom",
+    args: {
+        isOpened: false,
+    },
+    render: (args) => {
+        const [isOpen, setIsOpen] = useState(args.isOpened);
+
+        useEffect(() => {
+            setIsOpen(args.isOpened);
+        }, [args.isOpened]);
+
+        return (
+            <>
+                <Button
+                    size="lg"
+                    label={openModalButtonLabel}
+                    onClick={() => setIsOpen(true)}
+                />
+                <ModalBottom
+                    {...args}
+                    isOpened={isOpen}
+                    toggleModal={setIsOpen}
+                >
+                    <ModalBottom.Title>
+                        {titlePlaceHolderText}
+                    </ModalBottom.Title>
+                    <ModalBottom.Body>{shortTextContent}</ModalBottom.Body>
+                </ModalBottom>
+            </>
+        );
+    },
+};
+
+export const ModalBottomWithoutHandleBar: Story = {
+    name: "Modal Bottom without handle-bar (no swiping)",
+    args: {
+        isOpened: false,
+        showHandleBar: false,
+    },
+    render: (args) => {
+        const [isOpen, setIsOpen] = useState(args.isOpened);
+
+        useEffect(() => {
+            setIsOpen(args.isOpened);
+        }, [args.isOpened]);
+
+        return (
+            <>
+                <Button
+                    size="lg"
+                    label={openModalButtonLabel}
+                    onClick={() => setIsOpen(true)}
+                />
+                <ModalBottom
+                    {...args}
+                    isOpened={isOpen}
+                    toggleModal={setIsOpen}
+                >
+                    <ModalBottom.Title>
+                        {titlePlaceHolderText}
+                    </ModalBottom.Title>
```

Comment:
Is it possible to unite `ModalBottom.Title` and `ModalBottom.Image` logic under a single `ModalBottom.Header` component which would accept `image` and/or `title` as props?
The reason why I'm suggesting it is that there is not much logic in `ModalBottom.Title` to be worth separating :)

Reply:
I think you are right, no sense in separating them. Will update PR

Reply:
Oh, okay, we can't unite them) Image and Text should be in separate containers for classnames

Reply:
Latest status: unite image and title into the header component with new flags for controlling children